package.json

```diff
@@ -8,7 +8,8 @@
     "test": "karma start",
     "prepublish": "typings install && webpack && webpack --config webpack.vendor.config.js",
     "build-docs": "jsdoc -c jsdoc-conf.json",
-    "bower": "bower install"
+    "bower": "bower install",
+    "postinstall": "bower install"
```

Comment:
From the [npm docs](https://docs.npmjs.com/misc/scripts): `install, postinstall: Run AFTER the package is installed.`

Meaning this change would make `npm install manageiq-ui-components` break in projects that don't use `bower`..